.prettierignore

```diff
@@ -8,3 +8,4 @@ build
 artifacts
 cache
 .github
+voter-dapp
```

Comment:
what's the reason to ignore this package?

Reply:
oops! No reason, I was messing around with a few experimental things and accidentally left that change. Will fix.

Reply:
Fixed.